pkg/pod/entrypoint.go

```diff
@@ -133,6 +133,9 @@ func orderContainers(commonExtraEntrypointArgs []string, steps []corev1.Containe
 			"-termination_path", terminationPath,
 			"-step_metadata_dir", filepath.Join(runDir, idx, "status"),
 		)
+		if isSidecarLogsResultsEnabled == true {
```

Comment:
Instead of passing through the bool, you can just append this to `commonExtraEntrypointArgs` in pod.go (this feature doesn't have much to do with container ordering)